ReCaptchaFrontendUi/view/frontend/web/js/reCaptcha.js

```diff
@@ -55,8 +55,7 @@ define(
 
                 element.async = true;
                 element.src = 'https://www.google.com/recaptcha/api.js' +
-                    '?onload=globalOnRecaptchaOnLoadCallback&render=explicit' +
-                    (this.settings.rendering.lang ? '&hl=' + this.settings.rendering.lang : '');
+                    '?onload=globalOnRecaptchaOnLoadCallback&render=explicit';
```

Comment:
Looks lie we also need to update
https://github.com/magento/security-package/blob/1.0-develop/ReCaptchaUser/view/adminhtml/templates/recaptcha.phtml#L47

Reply:
Updated also for Admin panel